faq.adoc

```diff
@@ -96,7 +96,8 @@ Example response
 }
 
 --
-For more information on the filter and the endpoint, see https://github.com/iZettle/api-documentation/blob/master/authorization.adoc[OAuth2 API documentation].
+
+For more information on the filter and the endpoint, see xref:/authorization.md[OAuth] for more info.
```

Comment:
"For more information on the filter and the endpoint, see xref:/authorization.md[OAuth]".

Reply:
Thanks! Fixed it.